apps/box/src/screens/Users/UserHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Clipboard from '@react-native-clipboard/clipboard';
+import { CopyIcon } from '../../components';
+import { User } from '../../api/users';
+import {
+  FxBox,
+  FxButton,
+  FxSpacer,
+  FxText,
+} from '@functionland/component-library';
+import { Image } from 'react-native';
+
+type UserHeaderProps = {
+  userData: User;
+};
+export const UserHeader = ({ userData }: UserHeaderProps) => {
+  const copyHandler = () => {
+    Clipboard.setString(userData.decentralizedId);
+  };
```

Comment:
Could move out of function so it's not recreated every render: 

```typescript
const copyHandler = (value: string) => {
    Clipboard.setString(value);
  };
  ```
  
  `<FxButton onPress={copyHandler(userData.decentralizedId)} ...`

Reply:
done